### ISSUE TITLE
Empty model saved without running validation

### ISSUE THREAD

**Issue by razorsharpshady** [yidas/codeigniter-model#19]
at line 357 in Model.php 
` return ($returnData) ? $data : true;`
Why is empty $data is returned true?

This causes Model to save without running validations e.g
``` 
$this->load->model('yidas/Client_model', 'client');
$this->client->save();
```

**Comment by yidas** [comment]
Hi @razorsharpshady,

This issue has been fixed from [version 2.16.6](https://github.com/yidas/codeigniter-model/releases/tag/2.16.6).

Thanks for your advice.

**Comment by razorsharpshady** [comment]
Thanks for the fix, Good Job Nick
